src/sites/facebook.ts

```diff
@@ -14,18 +14,18 @@ const facebook = new Site({
       validateUrl: url => url.pathname === '/',
       requiredUserConfigKey: UserConfigKey.FacebookHideHomeFeed,
       injectCss: `
-        div[role="feed"] {
+        #ssrb_feed_start + div {
```

Comment:
doesn't work anymore @tobiasdalhof . at least for me it's 
`div[role="main"]{`

Reply:
Using `div[role="main"]` seems to work, but it has the unintended side effect of removing the create post form. To address this, we could consider using a list of selectors and prioritize them, with `div[role="main"]` as the last selector in case others fail